### DISCUSSION TITLE
Streamer mode requests

### DISCUSSION THREAD

**Discussion by brysondev** [NeotokyoRebuild/neo#759] [Ideas]
Mzync has opened a PR for a streamer mode (#745)

So far we intend on having:
- Obfuscate console printouts
- Detect OBS at game launch for cvar initial value
- Custom name file to replace player names

If you have any other suggestions for the streamer mode functionality, please leave 'em here.

**Comment by blaberry** [comment]
Dupe of https://github.com/NeotokyoRebuild/neo/issues/150

**Comment by brysondev** [comment]
https://github.com/NeotokyoRebuild/neo/issues/150
